src/less/components/button.less

```diff
@@ -414,25 +422,13 @@
 
         &.btn-primary,
         &.btn-secondary,
-        &.btn-link {
-            &:after {
-                border-color: lighten(@brand-secondary, 40%);
-                border-left-color: @brand-secondary;
-            }
-        }
-
-        &.btn-executive-destructive {
-            &:after {
-                border-color: hsla(0, 0%, 100%, 0.5);
-                border-left-color: @brand-error;
-            }
-        }
-
+        &.btn-link,
+        &.btn-executive-destructive,
         &.btn-guiding-destructive,
         &.btn-link-destructive {
             &:after {
-                border-color: @brand-error-light;
-                border-left-color: @brand-error;
+                border-color: lighten(@brand-secondary, 40%);
+                border-left-color: @brand-secondary;
             }
         }
 
```

Comment:
Executive-destructiv button has the wrong box-shadow color. Should be #EE5058
![image](https://user-images.githubusercontent.com/44967773/184604760-b0d2182d-3edd-4eca-bc38-6551b0ca5015.png)